From 82b437f743ee4be704e1fe2b2779a5c67a02f626 Mon Sep 17 00:00:00 2001
From: Dongjin Kim <tobetter@gmail.com>
Date: Wed, 17 Nov 2021 03:25:51 +0900
Subject: [PATCH 06/33] ODROID-COMMON: add '--symbol' to device tree compiler

This change is to create device tree node '__symbol__' to support device
treey overlay features, *.dtbo won't be applied without this node.

Signed-off-by: Dongjin Kim <tobetter@gmail.com>
Change-Id: I3b6102c925e871947f16e7c570563011101f5f86
---
 scripts/Makefile.lib | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/Makefile.lib b/scripts/Makefile.lib
index 9f69ecdd7977..84149c324d2c 100644
--- a/scripts/Makefile.lib
+++ b/scripts/Makefile.lib
@@ -290,7 +290,7 @@ quiet_cmd_gzip = GZIP    $@
 # DTC
 # ---------------------------------------------------------------------------
 DTC ?= $(objtree)/scripts/dtc/dtc
-DTC_FLAGS += -Wno-interrupt_provider
+DTC_FLAGS += -Wno-interrupt_provider --symbol
 
 # Disable noisy checks by default
 ifeq ($(findstring 1,$(KBUILD_EXTRA_WARN)),)
-- 
2.38.0

